src/main/java/com/jcabi/xml/ClasspathResolver.java

```diff
@@ -47,11 +46,8 @@
     public LSInput resolveResource(final String type, final String namespaceuri,
         final String publicid, final String systemid, final String baseuri) {
         LSInput input = null;
-        if (systemid != null) {
-            final InputStream stream = getClass().getResourceAsStream(systemid);
-            if (stream != null) {
-                input = new ClasspathInput(publicid, systemid, stream);
-            }
+        if (systemid != null && getClass().getResource(systemid) != null) {
```

Comment:
@longtimeago Yes, we do, in order to preserve the same logic as before, As you can see, if the ``stream`` that was passed to ``ClasspathInput`` was ``null``, then the ``input`` reference was also left ``null``. If I remove the check, some unit tests fail.